flow/layers/clip_rrect_layer.cc

```diff
@@ -18,6 +18,8 @@ void ClipRRectLayer::Preroll(PrerollContext* context, const SkMatrix& matrix) {
   if (child_paint_bounds.intersect(clip_rrect_.getBounds())) {
     set_paint_bounds(child_paint_bounds);
   }
+  context->size_hints->emplace_back(
+      SkISize::Make(ceil(clip_rrect_.width()), ceil(clip_rrect_.height())));
```

Comment:
Be careful that when there's a rect of [0.9, 0.9] x [1.1, 1.1], you may need size 2x2 ([0, 0] x [2, 2]) instead of 1x1 depending on how you're going to use it.